exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/partition/FindPartitionConditions.java

```diff
@@ -228,13 +236,16 @@ private boolean isHolisticExpression(RexCall call) {
     return false;
   }
 
+  protected boolean inputRefToPush(RexInputRef inputRef) {
```

Comment:
Can be made private. Should be placed under public methods.

Reply:
This is intentionally made to be 'protected' for future extension.
Right now, FindPartitionCondition use position based inputRef(using BitSet dirs) to mark which inputRef should be pushed. But in future, we may use name based policy to decide which one to push.